src/Kunstmaan/NodeBundle/Helper/NodeHelper.php

```diff
@@ -264,12 +265,12 @@ public function deletePage(Node $node, $locale)
         $this->deleteNodeChildren($node, $locale);
         $this->em->flush();
 
-        $this->eventDispatcher->dispatch(
+        $event = $this->eventDispatcher->dispatch(
             Events::POST_DELETE,
             new NodeEvent($node, $nodeTranslation, $nodeVersion, $page)
         );
 
-        return $nodeTranslation;
```

Comment:
BC?

Reply:
Currently I don't see how to do it without BC. We can modify later that we use a listener instead of handling the response in the controller..